automated/linux/ostree/ostree.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -e
+# SPDX-License-Identifier: GPL-2.0-only
+# Copyright (C) 2021 Foundries.io Ltd.
+
+# shellcheck disable=SC1091
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+RESULT_FILE="${OUTPUT}/result.txt"
+export RESULT_FILE
+OSTREE_HASH=""
+
```

Comment:
Can we set SKIP_INSTALL to false so its clear.

Reply:
Done